interfaces/console/public/svg/OnBoardingDynamic.tsx

```diff
@@ -41,6 +42,7 @@ const OnBoardingDynamic = ({
   isShowComponents = false,
   textColor = '#304445',
   selectedColor = '#304445',
+  simPoolIconColor = '#6F7979',
 }: IOnBoardingDynamic) => {
   const orgIconColor = orgName === 'Organization' ? '#6F7979' : selectedColor;
```

Comment:
Same here use the colors from the theme, or add this one if it doesn't already exist.